locales/locales/en.json

```diff
@@ -4381,5 +4381,35 @@
 	},
 	"faviconUseLegacyDesc": {
 		"message": "If you don't like the new orange Reddit tab icon, revert to using the old Reddit icon instead."
+	},
+	"qhideName": {
```

Comment:
Could you update these to `quarantineHideDesc`, `quarantineHideFlair`, etc? consistency makes for easier Ctrl-F'ing.